params.go

```diff
@@ -0,0 +1,33 @@
+package wotsp
+
+import (
+	"crypto"
+)
+
+// Opts groups the parameters required for WOTSP operations.
+//
+// Opts implements crypto.SignerOpts and crypto.DecrypterOpts.
+type Opts struct {
+	Mode    Mode
+	PubSeed []byte
```

Comment:
Sounds reasonable

Reply:
another consideration is to accept `*Opts` instead of `Opts` to allow passing `nil` as valid options. Opts could then implement methods to decide mode/address values, like such:

```golang
func (o *Opts) mode() Mode {
    if o == nil {
        return W16
    }
    return o.Mode
}
```